### PR TITLE
replace customScalars with customScalarTypes

### PR DESCRIPTION
**Pull request by nesskazu**
## What kind of change does this PR introduce?

update docs.
Replace customScalars with customScalarTypes on the usage with relay page.

## What is the current behavior?

Reflecting changes in relay v16.2.0

## Additional context
- https://github.com/facebook/relay/releases/tag/v16.2.0
- https://github.com/facebook/relay/tree/main/packages/relay-compiler

### CODE REVIEW

File: docs/usage_with_relay.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Modify your `relay.config.js` file to reflect the following:
    ```
 
    - `schemaConfig` tells the Relay compiler where to find the `nodeId` field on the `node` interface
-   - `customScalars` will improve Relay's type emission
+   - `customScalarTypes` will improve Relay's type emission
```
Comment (imor) [inline docs/usage_with_relay.md:35]: @nesskazu could you please add a note that in relay versions older than v16.2.0 it should be named `customScalars`.

Comment (nesskazu) [reply docs/usage_with_relay.md:35]: Thanks for the feedback. I've added the note. https://github.com/supabase/pg_graphql/pull/547/commits/74368fc4b1659b5e099e40271647f78c30bc7cb7

**Review by imor — APPROVED** [review]

**Comment by olirice** [comment]
pinged Alaister for approval. lets wait for that before merging

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**